gui/src/components/dashboard/cortx-dashboard-cluster-health-card.vue

```diff
@@ -1,76 +1,122 @@
-          :callBack="infoCardCallBack"
-          :imgUrl="cardDetail.imgUrl"
-          :key="index"
-        />
-      </template>
-    </div>
+    <cortx-card
+      :title="$t('dashboard.clusterHealth')"
+      :zoomIconCallback="zoomIconHandler"
+    >
+      <cortx-dashboard-info-card
+        title="mycluster"
+        :description="$t('dashboard.degraded')"
+        navPath="/health"
+        :callBack="infoCardCallBack"
+        :imgUrl="getClusterHealthImgUrl('degraded')"
+      />
+      <p class="cortx-text-lg cortx-text-bold">{{ $t("dashboard.nodes") }}</p>
+      <div class="node-health-cards-container">
+        <template v-for="(cardDetail, index) in nodeCardDetails">
+          <cortx-dashboard-info-card
+            :title="cardDetail.title"
+            :description="cardDetail.description"
+            :navPath="cardDetail.navPath"
+            :callBack="infoCardCallBack"
+            :imgUrl="cardDetail.imgUrl"
+            :key="index"
+          />
+        </template>
+      </div>
+    </cortx-card>
   </div>
 </template>
 <script lang="ts">
 import { Component, Vue } from "vue-property-decorator";
 import CortxDashboardInfoCard from "./cortx-dashboard-info-card.vue";
+import CortxCard from "./cortx-card.vue";
 
 @Component({
   name: "cortx-dashboard-cluster-health-card",
-  components: { CortxDashboardInfoCard }
+  components: { CortxDashboardInfoCard, CortxCard }
 })
 export default class CortxDashboardClusterHealthCard extends Vue {
-  public healthCardDetails = [
+  public nodeCardDetails = [
     {
-      title: "mycluster",
-      description: "Cluster",
-      imgUrl: "dashboard/health/cluster-health-warning.svg",
+      title: "5",
+      description: this.$t("dashboard.onlineNodes"),
+      imgUrl: this.getNodeImgUrl(5, "online"),
       navPath: "/health"
     },
     {
-      title: "05",
-      description: "Online Nodes",
-      imgUrl: "dashboard/health/online-nodes.svg",
+      title: "5",
+      description: this.$t("dashboard.offlineNodes"),
+      imgUrl: this.getNodeImgUrl(5, "offline"),
       navPath: "/health"
     },
     {
-      title: "05",
-      description: "Offline Nodes",
-      imgUrl: "dashboard/health/failed-nodes.svg",
+      title: "0",
+      description: this.$t("dashboard.failedNodes"),
+      imgUrl: this.getNodeImgUrl(0, "failed"),
       navPath: "/health"
     },
     {
-      title: "05",
-      description: "Failed Nodes",
-      imgUrl: "dashboard/health/failed-nodes.svg",
-      navPath: "/health"
-    },
-    {
-      title: "05",
-      description: "Degraded Nodes",
-      imgUrl: "dashboard/health/degraded-nodes.svg",
+      title: "5",
+      description: this.$t("dashboard.degradedNodes"),
+      imgUrl: this.getNodeImgUrl(5, "degraded"),
       navPath: "/health"
     }
   ];
+
   infoCardCallBack(routePath: string) {
     this.$router.push(routePath);
   }
+
+  getClusterHealthImgUrl(healthType: "offline" | "degraded" | "failed") {
+    switch (healthType) {
```

Comment:
can use a Json object instead of switch  eg {"offline":"dashboard/health/offline-cluster.svg", ...}

Reply:
Done